scripts/Interface/ui/drill/view/SettingsDialog.py

```diff
@@ -0,0 +1,261 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import os
+from qtpy.QtCore import QObject, Signal
+
+from qtpy.QtWidgets import QDialog, QComboBox, QCheckBox, QLineEdit, QLabel
+
+from qtpy import uic
+
+from mantidqt.widgets.filefinderwidget import FileFinderWidget
+from mantidqt.widgets.workspaceselector import WorkspaceSelector
+
+
+class Setting(QObject):
```

Comment:
Should we prefix this (and the file) with `Drill`?

Reply:
I renamed it and the associated tests.